designer/client/src/components/graph/graphTheme.styl

```diff
@@ -5,14 +5,16 @@ linkColor = white
 .arrow-marker path
     fill linkColor
 
-.joint-theme-default.touch
-    .joint-link
-        .link-tool
-        &:hover .connection-wrap
-            display none
-        .marker-arrowhead
-            opacity 0
-            transform scale(.3)
+@media (hover: none)
```

Comment:
We need to detect touch events on the CSS side, Passing custom className, in this case, broke the hybrid device node connection, and also events like panning, work "strange". That's because this custom class was added based on device type, not based on the current event